sormas-app/app/src/main/java/de/symeda/sormas/app/backend/customizableenum/CustomizableEnumValueDao.java

```diff
@@ -72,6 +72,32 @@ public String getTableName() {
 		return CustomizableEnumValue.TABLE_NAME;
 	}
 
+	@SuppressWarnings("unchecked")
+	public <T extends CustomizableEnum> List<T> getEnumValues(CustomizableEnumType type) {
```

Comment:
See above, use the method with the disease parameter instead and pass null as the disease